components/service/pocket/src/main/java/mozilla/components/service/pocket/stories/db/PocketRecommendationsDatabase.kt

```diff
@@ -32,9 +44,31 @@ internal abstract class PocketRecommendationsDatabase : RoomDatabase() {
                 PocketRecommendationsDatabase::class.java,
                 DATABASE_NAME
             )
+                .addMigrations(
+                    Migrations.migration_1_2,
+                )
                 .build().also {
                     instance = it
                 }
         }
     }
 }
+
+internal object Migrations {
+    val migration_1_2 = object : Migration(1, 2) {
+        override fun migrate(database: SupportSQLiteDatabase) {
+            database.execSQL(
+                "CREATE TABLE IF NOT EXISTS " +
+                    "`${PocketRecommendationsDatabase.TABLE_NAME_SPOCS}` (" +
+                    "`url` TEXT NOT NULL, " +
+                    "`title` TEXT NOT NULL, " +
+                    "`imageUrl` TEXT NOT NULL, " +
+                    "`sponsor` TEXT NOT NULL, " +
+                    "`clickShim` TEXT NOT NULL, " +
+                    "`impressionShim` TEXT NOT NULL, " +
+                    "PRIMARY KEY(`url`)" +
+                    ")"
+            )
+        }
+    }
+}
```

Comment:
@Mugurell could we have instrumentation tests just to make sure everything behave as expected, some examples can be found [here](https://github.com/mozilla-mobile/android-components/blob/main/components/feature/sitepermissions/src/androidTest/java/mozilla/components/feature/sitepermissions/db/OnDeviceSitePermissionsStorageTest.kt#L74-L108).

Reply:
I noticed with have [PocketRecommendationsDatabaseTest](https://github.com/mozilla-mobile/android-components/pull/12070/files#diff-1bb5a518761ef7a6a64fc2dd1bc036994a6b51a0f152db19e2b5510235783786R22), as it's an unit tests and not an instrumentation tests I'm not sure if we are going to be able to execute all room db integrity validations.

Reply:
To me this approach seems like an improvement. It's the same test just running much faster in the JVM rather than on a real device.